package.json

```diff
@@ -64,6 +65,9 @@
     "@changesets/changelog-github": "0.5.0",
     "@changesets/cli": "2.27.1",
     "@theguild/prettier-config": "2.0.6",
+    "@tsconfig/node16": "^16.1.3",
```

Comment:
Perhaps move to Node 18 or 20 ? 16 is no longer maintained.

Reply:
Node 16 is the lowest node engine in all packages. Keeping it for now until we upgrade.